hw2/05-form.js

```diff
@@ -0,0 +1,32 @@
+const form = document.querySelector("form");
+const nameInput = document.querySelector("#name");
+const emailInput = document.querySelector("#email");
+const registrationStatusInput = document.querySelector("#registration-status");
+const fullTimeCourseInputs = document.querySelectorAll('input[name="full-time-course"]');
+const additionalInfoInput = document.querySelector("#additional-info");
+
+form.addEventListener("submit", function (event) {
+  event.preventDefault();
+
+  const formData = {
+    name: nameInput.value,
+    email: emailInput.value,
+    registrationStatus: registrationStatusInput.value,
+    fullTimeCourse: [],
+    additionalInfo: additionalInfoInput.value
+  };
```

Comment:
I think it's better to use the `event` to take the data out instead or else it kind of defeat the purpose of having the form value in `event`